src/pages/community-tools.astro

```diff
@@ -1,5 +1,5 @@
 ---
-import MainLayout from "@/layouts/MainLayout.astro";
```

Comment:
isnt it equal? was the change needed?

Reply:
You're right. Same applies to the other imports that used ../ instead of @/ Fixed with the latest update.